projects/ngrx.io/content/guide/component-store/comparison.md

```diff
@@ -15,7 +15,7 @@ nodes of the components tree?
 some pages are displayed and we want to automatically clean it up when the user navigates somewhere
 else?
 * **Business Requirements**. How well are all of the business requirements understood and finalized before the
-implementation of the app starts? Would the be changing frequently?
+implementation of the app starts? Would it be frequently changing?
```

Comment:
```suggestion
implementation of the app starts? Would it be changing frequently?
```
WDYT about that one? I'd like to have an emphasis on `frequently`. :)

Reply:
Thanks for the PR BTW 👍

Reply:
Happy to help, and yes let's go with the emphasis on frequently👌